### ISSUE TITLE
[Bug] Telephone component 711 and 911 phone number links may not dial correctly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#338]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## Severity
**If someone can recreate these results on Staging or Prod, we should bump this to a defect 1 item for accessibility and fix as soon as possible.**

## What happened
I asked in a Slack thread if the [Telephone component](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/Telephone.jsx) links for `911` and `711` should have the extra "+1" in their `href` attribute. So I mocked up a test on my local machine to find out.

I made the following assumptions in local testing:
1. I didn't want to call the Telecommunications Relay Services 711 or 911 local emergency services numbers by accident
1. I used the National Digger's Hotline 811 as a reasonable proxy
1. I mocked up an HTML page and served it from a small Express server
1. I tried to complete a call using each of the three markup patterns for 811
1. I added Apple's certified 800 number as a smoke test to ensure calls were being completed at all

## What I expected to happen
1. I expected the call to go through to the Digger's Hotline

## Reproducing
- Formation version: latest
- Device: iPhone 8, Verizon
- Browser: iOS Safari

Steps to reproduce:
1. Copy the code snippet below into an HTML file
1. Serve the file using a local webserver like Python *http.server* or *http-server*
1. Load the webpage on your phone
1. Press to make a phone call.
  * The first option will complete, but presents strange visually.
  * The second option completes correctly.
  * The third option won't complete as dialed.

```html
<main aria-labelledby="heading-one">
  <h1 id="heading-one">Telephone link formatting for special numbers</h1>
  <p>Special numbers like 711 or 911 must be coded correctly to work as links. Since these are not easy to test without accessing Telecommunications Relay Service or emergency services, I'm using the National Digger's Hotline 811 number as a resonable proxy.</p>
  <div>
    <ul>
      <li>
        <p>National Digger's Hotline <a href="tel:811;phone-context=+1" aria-label="8 1 1.">811</a> with <code>;phone-context=+1</code></p>
        <p>Result: Phone call goes through on iOS, but the phone number on screen doesn't present correctly. Potentially confusing.</p>
      </li>
      <li>
        <p>National Digger's Hotline <a href="tel:811" aria-label="8 1 1.">811</a> without phone context</p>
        <p>Result: Call goes through as expected. This is the best candidate recommendation.</p>
      </li>
      <li>
        <p>National Digger's Hotline <a href="tel:+1811" aria-label="8 1 1.">811</a> with 1</p>
        <p>Result: This call cannot be completed. This pattern must be avoided.</p>
      </li>
      <li>
        <p><a href="tel:+18006927753" aria-label="8 0 0. 6 9 2. 7 7 5 3.">Apple Computers, toll free</a></p>
        <p>Result: This number adds some extra visuals because it's verified.</p>
      </li>
    </ul>
  </div>
</main>
```

```diff
! The +1 should be removed, so the href is just "711", "811", "911"

- <a href="tel:+1811" aria-label="8 1 1.">811</a>
+ <a href="tel:811" aria-label="8 1 1.">811</a>
```

## Screen shots
![IMG-2508](https://user-images.githubusercontent.com/934879/99125434-a57cb600-25c9-11eb-85ae-940d61891e20.PNG)

---
![IMG-2509](https://user-images.githubusercontent.com/934879/99125636-17ed9600-25ca-11eb-9d53-9a49ede00de5.PNG)

---
![IMG-2510](https://user-images.githubusercontent.com/934879/99125462-b4fbff00-25c9-11eb-8b21-ea2c621e4c72.PNG)

**Comment by 1Copenut** [comment]
Related issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/218

**Comment by bkjohnson** [comment]
So are you saying that all 3-digit numbers shouldn't have the `+1` prepended, but a typical phone number should?

**Comment by 1Copenut** [comment]
In testing, 811 (Nat'l Digger's Hotline) wouldn't complete my call as dialed with a `+1` prepended. @Mottie also tried our current `711` link and it wouldn't complete the call as dialed. So yes, I'm saying our three-digit numbers `['711', '911']` need to have the href refactored without the `+1`. I'll update the description for clarity.

/cc @jenstrickland

**Comment by jenstrickland** [comment]
Thanks, @1Copenut! Good to see the logic we discussed panned out as expected. 🎉

**Comment by Mottie** [comment]
All fixes have been merged and should be available in staging soon.

**Comment by 1Copenut** [comment]
Closing this issue per @Mottie work on [component library PR #36](https://github.com/department-of-veterans-affairs/component-library/pull/36) and verifying the fix by spot checking the Telephone component in staging.

**Comment by 1Copenut** [comment]
Finished verifying this issue was closed on 7/26. Closed the issue 7/27.
